scripts/background.js

```diff
@@ -145,85 +157,120 @@ function processHeaders(details) {
             }
         }
 
-        // 403 with the right header indicates a bypassable CAPTCHA
-        if (isBypassHeader(header) && details.statusCode == 403) {
-            doRedeem = true;
+        // correct status code with the right header indicates a bypassable Cloudflare CAPTCHA
+        if (isBypassHeader(header) && SPEND_STATUS_CODE.indexOf(details.statusCode) > -1) {
+            let iframe = (details.frameId > 0);
+            // check if the token should only be spent on an iframe
+            if (SPEND_IFRAME) {
```

Comment:
```suggestion
            if ((SPEND_IFRAME && iframe) || !SPEND_IFRAME) {
                activated = true
            }
```
No need to set `activated` to false here--that's already its default.

Reply:
Yeah you're right, will fix.